libbeat/tests/system/beat/beat.py

```diff
@@ -350,6 +353,16 @@ def output_has(self, lines, output_file=None):
         except IOError:
             return False
 
+    def output_has_message(self, message, output_file=None):
```

Comment:
Yeah, I tried working with the existing APIs but checking the length wasn't enough since when I restart I want to check for the different message to show up (length still 1)